docs/getting-started.md

```diff
@@ -0,0 +1,98 @@
+
+## 5. Configure a strong password for the default `admin` user on the Harvester UI. 
+
+Once the cluster status changes to **Ready**, you can access the [Harvester UI](../authentication) using the management URL displayed on the console. 
+
+## 6. Deploy the other nodes and join them to the cluster. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. All other nodes join the cluster that was created when the first node was deployed. 
+
+During installation, you must configure node settings, and specify the cluster management address (virtual IP) and the cluster token that you defined previously. If necessary, you can configure more settings using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+When the cluster has three or more nodes, the two nodes added after the first node are automatically promoted to management nodes to form a high-availability (HA) cluster. 
+
+## 7. Create a cluster network and a VM network.
+
+Networking in Harvester involves three major concepts:
+
+- [**Cluster network**](../networking/index#cluster-network): Traffic-isolated forwarding path for transmission of network traffic in the Harvester cluster. 
+    
+    During deployment, a cluster network called the [*management network*](../networking/index#built-in-cluster-network) is created for intra-cluster communications. Harvester allows you to create [custom cluster networks](../networking/index#custom-cluster-network) that can be dedicated to VM traffic and that allow VMs to be accessed from both internal and external networks.
+
+- [**Network configuration**](../networking/index#cluster-network): Definition of how cluster nodes connect to a specific cluster network. 
+    
+    Each network configuration corresponds to a set of nodes with uniform network specifications. Only nodes that are covered by the network configuration can access the associated cluster network.
+
+- [**VM network**](../networking/index#vm-network): Virtual network that VMs use to communicate with other VMs and external networks.
+    
+    Each VM network is linked to a specific cluster network, which is used for transmission of VM traffic. You can create either a [VLAN network](../networking/harvester-network/#vlan-network) or an [untagged network](../networking/harvester-network/#untagged-network) based on your requirements, such as traffic isolation, network segmentation, ease of management, or alignment with the external network environment. 
+
+Before you create VMs, [create a custom cluster network](../networking/index#how-to-create-a-new-cluster-network) (including the associated network configuration), and then [create a VM network](../networking/index#how-to-create-a-new-cluster-network) that is linked to the custom cluster network.
```

Comment:
Maybe we can mention that if the user has more than one network interface per node, they can create additional custom cluster networks and network configurations. If not, they can only use the management network to create VM networks.

Reply:
Thank you. PTAL at the updated text.